syntax = "proto2";

package proto.db;

message update_command {
	required string message_name = 1;
	required bytes message_content = 2;
}